SimPEG/regularization/base.py

```diff
@@ -54,19 +54,44 @@ def __init__(
                 f"Value of type {type(mesh)} provided."
             )
 
+        # Handle deprecated indActive argument
+        if (key := "indActive") in kwargs:
```

Comment:
You know, this is the first time I've seen the `:=` expression assignment operator used, (even though it's been around since 3.8)!

Reply:
I've started using it not long ago... When we support older versions of Python we have to wait to use the new shiny stuff ([`match ... case`](https://peps.python.org/pep-0636/), I'm looking at you).